src/utils/status/host/hostStatus.js

```diff
@@ -0,0 +1,6 @@
+import { HOST_STATUS_READY } from './constants';
+
+// TODO(honza): Fill this out
+export const getHostStatus = host => ({
+  status: HOST_STATUS_READY,
```

Comment:
@flofuchs How could we use your work in #293 here?

Reply:
yup, it would make sense to introduce this method according to our conventions so we can  use it in inventory and other places